public/src/pages/main/Footer.tsx

```diff
@@ -151,22 +158,35 @@ export const Footer: FC = () => {
               <InnerHTML html={decodeURIComponent(window.location.search)} />
             </span>
           </div>
-          <div className="social-links text-center text-md-right pt-3 pt-md-0">
-            <a href="/" className="twitter">
-              <i className="bx bxl-twitter" />
-            </a>
-            <a href="/" className="facebook">
-              <i className="bx bxl-facebook" />
-            </a>
-            <a href="/" className="instagram">
-              <i className="bx bxl-instagram" />
-            </a>
-            <a href="/" className="google-plus">
-              <i className="bx bxl-skype" />
-            </a>
-            <a href="/" className="linkedin">
-              <i className="bx bxl-linkedin" />
-            </a>
+          <div>
+            <div className="social-links text-center text-md-right pt-3 pt-md-0">
+              <a href="/" className="twitter">
+                <i className="bx bxl-twitter" />
+              </a>
+              <a href="/" className="facebook">
+                <i className="bx bxl-facebook" />
+              </a>
+              <a href="/" className="instagram">
+                <i className="bx bxl-instagram" />
+              </a>
+              <a href="/" className="google-plus">
+                <i className="bx bxl-skype" />
+              </a>
+              <a href="/" className="linkedin">
+                <i className="bx bxl-linkedin" />
+              </a>
+            </div>
+            <div className="social-links text-center text-md-right pt-md-2">
+              Supporting:{' '}
+              {cloudProviders.map((provider) => (
+                <a
+                  href={`/api/file/${provider.name}?path=${provider.name}`}
+                  className="google"
```

Comment:
Should the class name correlate with the cloud provider name?

Reply:
@derevnjuk actually it serves no purpose there at all. Removing :)